orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/restore_db.sh

```diff
@@ -57,9 +56,15 @@ errorExit() {
 kinit_as_hdfs() {
   # Use a new Kerberos credential cache, to keep from clobbering the default.
   export KRB5CCNAME=/tmp/krb5cc_cloudbreak_$EUID
-  HDFS_KEYTAB=/run/cloudera-scm-agent/process/$(ls -t /run/cloudera-scm-agent/process/ | grep hdfs-NAMENODE$ | head -n 1)/hdfs.keytab
-  doLog "kinit as hdfs using Keytab: $HDFS_KEYTAB"
-  kinit -kt "$HDFS_KEYTAB" "hdfs/$(hostname -f)"
+
+  KEYTAB=$(find /run/cloudera-scm-agent/process/ -name "*.keytab" -path "*hdfs*" -o \
+   -path "*hbase-REGIONSERVER" -o \
+   -path "*hbase-MASTER" -o \
+   -path "*solr-SOLRSERVER" \
+   | head -n 1)
+
+  doLog "kinit as hdfs using Keytab: $KEYTAB"
+  kinit -kt "$KEYTAB" "hdfs/$(hostname -f)"
```

Comment:
With the above change, you might use the key tab for hdfs/hbase/solr services but when you perform kinit you are always using hdfs as the principal name.